src/Public/LogRhythm/Case/Test-LrPlaybookIdFormat.ps1

```diff
@@ -0,0 +1,63 @@
+using namespace System
+using namespace System.IO
+using namespace System.Collections.Generic
+
+Function Test-LrPlaybookIdFormat {
```

Comment:
Since PlayBook Id can ONLY be a `guid`, this function can be removed.  

You can use `Test-Guid` anywhere you currently use `Test-LrPlaybookIdFormat`.  The only result of `Test-LrPlaybookIdFormat` would be guid is true or guid is false.  :)

Note: The reason I have `Test-LrCaseIdFormat` is because cases have two kinds of ID - guid and number.  Both are valid ways to look up a unique case, and I wanted to allow the user to specify either with the same parameter.

Reply:
Deleted